voila/handler.py

```diff
@@ -150,6 +150,7 @@ def time_out():
 
                 return '<script>voila_heartbeat()</script>\n'
 
+            kernel_env = {**os.environ.copy(), **request_info}
```

Comment:
```suggestion
            kernel_env = {**os.environ, **request_info}
```
I don't think the copy adds anything, since we already create a new dict